src/components/TodoItemSort/TodoItemSort.js

```diff
@@ -0,0 +1,48 @@
+import React, { Component } from 'react';
+import classNames from 'classnames';
+import { TodoItemSortProps } from '../../constants/proptypes';
+
+import './TodoItemSort.css';
+
+class TodoItemSort extends Component {
+  buttons = [
+    { name: 'default', label: 'Default' },
+    { name: 'title', label: 'Title' },
+    { name: 'status', label: 'Status' },
+    { name: 'user', label: 'User' },
+  ];
+
+  render() {
+    const { sort, onSortChange } = this.props;
+
+    const buttons = this.buttons.map(({ name, label }) => {
+      const isActive = sort === name;
+
+      const classes = classNames({
```

Comment:
give more specific name instead of `classes`

Reply:
fixed